litestar/_openapi/path_item.py

```diff
@@ -1,141 +1,137 @@
+
+        return self._path_item
+
+    def create_operation_for_handler_method(
+        self, route_handler: HTTPRouteHandler, http_method: HttpMethod
+    ) -> Operation:
+        """Create an Operation instance for a given route handler and http method.
+
+        Args:
+            route_handler: A route handler instance.
+            http_method: An HttpMethod enum value.
+
+        Returns:
+            An Operation instance.
+        """
+        operation_id = self.create_operation_id(route_handler, http_method)
+        parameters = create_parameters_for_handler(self.context, route_handler, self.route.path_parameters)
+        signature_fields = route_handler.parsed_fn_signature.parameters
+
+        request_body = None
+        if data_field := signature_fields.get("data"):
+            request_body = create_request_body(
+                self.context, route_handler.handler_id, route_handler.resolve_data_dto(), data_field
+            )
+
+        raises_validation_error = bool("data" in signature_fields or self._path_item.parameters or parameters)
+        responses = create_responses_for_handler(
+            self.context, route_handler, raises_validation_error=raises_validation_error
+        )
+
+        return route_handler.operation_class(
+            operation_id=operation_id,
+            tags=route_handler.resolve_tags() or None,
+            summary=route_handler.summary or SEPARATORS_CLEANUP_PATTERN.sub("", route_handler.handler_name.title()),
+            description=self.get_description_for_handler(route_handler),
+            deprecated=route_handler.deprecated,
+            responses=responses,
+            request_body=request_body,
+            parameters=parameters or None,  # type: ignore[arg-type]
+            security=route_handler.resolve_security() or None,
+        )
+
+    def create_operation_id(self, route_handler: HTTPRouteHandler, http_method: HttpMethod) -> str:
+        """Create an operation id for a given route handler and http method.
+
+        Adds the operation id to the context's operation id set, where it is checked for uniqueness.
+
+        Args:
+            route_handler: A route handler instance.
+            http_method: An HttpMethod enum value.
+
+        Returns:
+            An operation id string.
+        """
+        if isinstance(route_handler.operation_id, str):
+            operation_id = route_handler.operation_id
+        elif callable(route_handler.operation_id):
+            operation_id = route_handler.operation_id(route_handler, http_method, self.route.path_components)
+        else:
+            operation_id = self.context.openapi_config.operation_id_creator(
+                route_handler, http_method, self.route.path_components
+            )
+        self.context.add_operation_id(operation_id)
+        return operation_id
+
+    def get_description_for_handler(self, route_handler: HTTPRouteHandler) -> str | None:
```

Comment:
```suggestion
    def create_description_for_handler(self, route_handler: HTTPRouteHandler) -> str | None:
```

Just to maintain consistency with the other methods that are named `create_*`.